lib/authorizer/auth-interface.js

```diff
@@ -109,6 +110,30 @@ createAuthInterface = function (auth, protocolProfileBehavior, options) {
             });
 
             return this;
+        },
+
+        /**
+         * @param {String} type - type of auth
+         * @return {AuthState|undefined}
+         */
+        getAuthState: function (type) {
```

Comment:
Since this is auth-interface, using `auth` in the name is redundant. It can simply be `getState` and `setState`.